k8s/pkg/apis/longhorn/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -146,6 +146,67 @@ func (in *EngineList) DeepCopyObject() runtime.Object {
 	return nil
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
Separates added code and generated code.

Reply:
Separating manually added code versus generated code.